### DISCUSSION TITLE
What is the Middle Eastern egg dish baked into tomatoes and generally eaten for breakfast?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#556] [Galaxy Trivia]
**Bonus:** Describe the recipe using as many of the emojis as you can.

**Comment by peterbaccaro** [comment]
**Shakshuka** 🍅🥚🧅🧄🧂

**Comment by SebastienXia** [comment]


**Reply by ParanoidUser** [reply]
Move over boring old eggs, Feta just showed up and took your shakshuka game to the next level!

Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/SebastienXia?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Reply by peterbaccaro** [reply]
Feta is not part of the traditional recipe though. 😝

**Comment by thuongtruong1009** [comment]
The Middle Eastern egg dish that is baked into tomatoes and generally eaten for breakfast is called Shakshuka. It is a popular dish in Middle Eastern and North African cuisine, consisting of poached eggs in a spicy tomato sauce that is flavored with peppers, onions, and various spices such as cumin and paprika. It is often served with bread or pita for dipping in the sauce. Shakshuka is a hearty and flavorful dish that is enjoyed around the world, particularly as a brunch or breakfast item.
